README.md

```diff
@@ -140,6 +140,36 @@ The `document` function gives developers a quick and easy way to document the va
 
 The documentation provided via this function is collected and exposed via the `describe` command detailed below.
 
+#### `add_alias`
+
+The `add_alias` function performs double duty in that is will a) add the alias you specify, and b) automatically document that the alias exists.
```

Comment:
[typo] The `add_alias` function performs double duty, it will a) ..., and b) ...